tests/jobs/rft/test_gendata_rft.py

```diff
@@ -0,0 +1,219 @@
+import os
+from distutils.dir_util import copy_tree
+import numpy.testing as npt
+import pytest
+
+from semeio.jobs.scripts.gendata_rft import main_entry_point
+
+ECL_BASE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "data/norne/NORNE_ATW2013"
+)
+EXPECTED_RESULTS_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "data/norne/expected_result"
+)
+
+MOCK_DATA_CONTENT = {
+    "RFT_B-1AH_0_active": [1] * 5,
+    "RFT_B-4AH_0_active": [1] * 7 + [0] * 1 + [1] * 21,
+    "RFT_B-4H_0_active": [1] * 20,
+    "RFT_C-1H_0_active": [1] * 21,
+    "RFT_C-3H_0_active": [1] * 21,
+    "RFT_C-4AH_0_active": [1] * 15,
+}
+
+
+def _assert_almost_equal_line_by_line(file1, file2):
+    with open(file1) as fh:
+        file1_content = fh.readlines()
+
+    with open(file2) as fh:
+        file2_content = fh.readlines()
+
+    assert len(file1_content) == len(file2_content)
+
+    for line1, line2 in zip(file1_content, file2_content):
+        try:
+            line1, line2 = float(line1), float(line2)
+        except ValueError:
+            continue
+        npt.assert_almost_equal(line1, line2, decimal=7)
```

Comment:
can we just use  `numpy.testing.asser...`? it us only being used once.

Reply:
agreed, fixing